electrum_nmc/electrum/tests/test_util.py

```diff
@@ -122,14 +92,10 @@ def test_parse_URI_invalid_address(self):
         self.assertRaises(BaseException, parse_URI, 'namecoin:invalidaddress')
 
     def test_parse_URI_invalid(self):
-        #self.assertRaises(BaseException, parse_URI, 'notbitcoin:15mKKb2eos1hWa6tisdPwwDC1a5J1y9nma')
-        # Converted to Namecoin using `contrib/convertAddress.py` from Namecoin Core.
-        self.assertRaises(BaseException, parse_URI, 'notnamecoin:N1LgXEXdjF7G37MPzgwyATN6joULz6LfdQ')
+        self.assertRaises(BaseException, parse_URI, frombtc('invalidprotocol:15mKKb2eos1hWa6tisdPwwDC1a5J1y9nma'))
```

Comment:
Why `invalidprotocol:`?  Seems to me that we should be making `parse_URI` receive a protocol of `notnamecoin:`, so that we'll be sure to catch bugs where it accepts invalid protocols whose substring is a valid protocol.  (Seems like an edge case that's unlikely to be hit, but doesn't hurt to test for it.)

Reply:
(Which means that `from_btc` might need tweaks to properly convert substrings of the protocol.)

Reply:
I just chose `invalidprotocol` because that is the "obvious" intent of the test (i.e. what happens for a protocol that is not the expected one).  Your point about it containing "namecoin" as a substring is valid, though, so I've just changed it back to `notnamecoin:`.  I think we can just explicitly change this one (and let `frombtc` just convert the address), instead of adding a special case.